mlir/lib/Target/SPIRV/Deserialization/Deserializer.cpp

```diff
@@ -2158,13 +2158,50 @@ LogicalResult spirv::Deserializer::wireUpBlockArgument() {
   return success();
 }
 
+LogicalResult spirv::Deserializer::splitConditionalBlocks() {
+  for (auto it = blockMergeInfo.begin(); it != blockMergeInfo.end(); it++) {
+    // Skip processing loop regions. For loop regions continueBlock is non-null.
+    if (it->second.continueBlock)
```

Comment:
you can do `auto& [x, y] = *it` to give `first` and `second` some meaningful names.